.vscode/launch.json

```diff
@@ -15,7 +15,8 @@
       "stopAtEntry": false,
       "serverReadyAction": {
         "action": "openExternally",
-        "pattern": "\\bNow listening on:\\s+(https?://\\S+)"
+        "pattern": "\\bNow listening on:\\s+(https?://\\S+)",
+        "uriFormat": "%s/swagger"
```

Comment:
is this needed?

Reply:
not strictly needed, but makes "F5-Debug" land on an actual page